session.json

```diff
@@ -0,0 +1 @@
+{"macKey": "\u00dc]\u00c4\u00aaG\f\u0003G\u00fa\u00a3.\u00e6xK;\u00e6\u00de}y\u001e\u0017\u00ae\u0017\u00d3\rH\u00c1R\u0004\u00a4\u0019\u0011", "serverToken": "1@B6whfXk6pYOR7Yt6GjLoQST3RfXAZpp19dsQz3aUvKo87z7Xpw5ceI2ZbeApDaeu+DjohY+ZqMOVJQ==", "encKey": "\u00e4\u0083In\u007f1\u001f\u00f1\n\u00965\u001c\u00cbn\u0085hM<\u0082\n\u001dT\u00f4\u00b5\u00b9y\u00ee\u00f3{\u0085\bN", "clientId": "5svFWIYYQdBGTxqFyhcsUg==", "clientToken": "InUzqzrXdq5ZwqLHwMMjNSmsV3ZzTHLmHwLPvh1YiME="}
```

Comment:
Did you mean to commit this file?

Reply:
hi quad.
just want to be clear how code works, because many people ask about how whatsapp reveng works.and the session that stored in this file is expired. you can remove it from merge requst